### DISCUSSION TITLE
Welcome to FioriAR Discussions!

### DISCUSSION THREAD

**Discussion by MarcoEidinger** [SAP-archive/cloud-sdk-ios-fiori-ar#72] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by baotrungas99** [comment]
I've got a problem when I try to build an example screen fioriar into my physical device and I don't know how to fix it. Can you help me find down what's wrong I am? thanks in advance. 
<img width="1790" alt="Screenshot 2023-09-18 at 23 11 13" src="https://github.com/SAP/cloud-sdk-ios-fiori-ar/assets/54469441/b75d134f-057f-428f-ba2b-22d4fa38a029">

**Reply by MarcoEidinger** [reply]
@baotrungas99 please use [issues](https://github.com/SAP/cloud-sdk-ios-fiori-ar/issues) to get help for technical issues. Your error looks similar to the one reported in https://github.com/SAP/cloud-sdk-ios-fiori-ar/issues/83, which got fixed with version 3.0.0. If you do use that version and face still an issue then please create an issue with detailed information to help us to help you. Example: which version you use, your physical device and it's iOS version, etc.
